modules/sqs/main.tf

```diff
@@ -47,13 +57,15 @@ data "aws_iam_policy_document" "status_update_queue_policy" {
 ## ==================================================================================================================
 resource "aws_sqs_queue" "internal_report_queue" {
   ## OPTIONAL
-  name                       = "${var.prefix}-orca-internal-report-queue"
-  delay_seconds              = var.sqs_delay_time_seconds
-  max_message_size           = var.sqs_maximum_message_size
-  message_retention_seconds  = var.internal_report_queue_message_retention_time_seconds
-  tags                       = local.tags
-  policy                     = data.aws_iam_policy_document.internal_report_queue_policy.json
-  visibility_timeout_seconds = 30 # Does not cover maximum processing time. Reconsider when multiple consumers becomes possible.
+  name                        = "${var.prefix}-orca-internal-report-queue.fifo"
+  fifo_queue                  = true
```

Comment:
Per our discussion on FIFO and how a FIFO DLQ we think will break the issue where processing would stop because order must be maintained, will leave as FIFO for now until we see some real world use/testing.